package api.techchallenge.domain.exception;

public class RecursoNaoEncontratoException extends RuntimeException {
    public RecursoNaoEncontratoException(String message) {
        super(message);
    }
}
